sources/packages/api/src/route-controllers/student-account-applications/models/student-account-application.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { Allow, IsObject } from "class-validator";
+import { CreateStudentAPIInDTO } from "../../../route-controllers/student/models/student.dto";
+
+export class CreateStudentAccountApplicationAPIInDTO {
+  @IsObject()
```

Comment:
We can use `@IsNotEmptyObject()` instead right?

Reply:
Changed.